### ISSUE TITLE
Can't apply two effects at once on triggers?

### ISSUE THREAD

**Issue by Ameno-Sagiri** [Paliverse/DualSenseX#102]
Unsure if a bug or intended.

Using Dualsense via bluetooth on my desktop. I can't seem to get DualSenseX to apply both "Vibrate Trigger Custom Intensity" AND "Resistance" at the same time. I found out about this software via Youtube and the tutorials I seen on there show them applying both vibrate trigger and resistance effects, stacking them with one another, so the trigger has a selectable resistance value and vibration value at the same time. I was intending to use this for racing games, but I can only seem to apply one trigger effect or the other, not both.

In addition to this problem, I have another bug(?) where even though I apply a controller mod/effect to BOTH triggers, it tends to only like applying it to the left trigger, and not the right trigger, but this happens at complete random and restarting the software seems to fix it temporarily. This is a nearly brand new controller with less than 5 hours of use, so I was assuming the issue isn't the controller itself.

I have also tested that even if I select NOTHING in the "Apply Mode" section, and click Apply, it will still apply it to either both triggers, or just the left trigger at random.

**Comment by Paliverse** [comment]
Hello, currently they can’t be stacked at all. What you probably saw was the “Automatic Gun” Trigger which has the strength slider (Resistance) and the frequency slider (Intensity) as well as the position where the effect is applied.

in terms of your bug, currently the app is undergoing complete redesign which has the Left Trigger and Right Trigger completely separated to make it easier. Here’s a preview of what’s to come by next week:

![A20F0E44-31A3-4FF6-93EC-E43D61E34410](https://user-images.githubusercontent.com/4289084/149687742-fdb9f28f-42e7-429b-8a01-2d6b7b1fed9d.jpeg)

**Comment by Ameno-Sagiri** [comment]
Thank you for replying, hopefully it will resolve the issue I was having. As for an update on my perceived problem, I just turned on my computer after yesterday and DualSenseX automatically started with Windows. It re-applied the modes to the triggers, but it applied the wrong frequency to the left trigger (I had set it to a form of  'VibrateResistance A' under Custom Trigger Values, but it seems to be vibrating at a much higher frequency than I had set), and I found it didn't apply Automatic Gun to the right trigger at all, which is what I ended up using yesterday.

It's not letting me apply Automatic Gun to the right trigger however. It seems I have to set the right trigger to random things I cycle through in the list before it lets me actually set it to what I want
